src/main/java/uk/gov/hmcts/cmc/claimstore/rules/CountyCourtJudgmentRule.java

```diff
@@ -87,4 +88,32 @@ public void assertRedeterminationCanBeRequestedOnCountyCourtJudgement(Claim clai
         }
 
     }
+
+    public boolean isCCJDueToSettlementBreach(Claim claim) {
+        requireNonNull(claim, "claim object can not be null");
+
+        if (claim.getSettlement().isPresent()) {
+            PaymentIntention paymentIntention = claim.getSettlement().orElseThrow(IllegalArgumentException::new)
+                .getLastOfferStatement().getOffer().orElseThrow(IllegalArgumentException::new)
+                .getPaymentIntention().orElseThrow(IllegalArgumentException::new);
+
+            switch (paymentIntention.getPaymentOption()) {
+                case IMMEDIATELY:
+                    return nowInLocalZone().toLocalDate().isAfter(
+                        paymentIntention.getPaymentDate().orElse(nowInLocalZone().toLocalDate()));
+                case BY_SPECIFIED_DATE:
+                    return nowInLocalZone().toLocalDate().isAfter(
+                        paymentIntention.getPaymentDate().orElse(nowInLocalZone().toLocalDate()));
+                case INSTALMENTS:
+                    return nowInLocalZone().toLocalDate().isAfter(
+                        paymentIntention.getRepaymentPlan()
+                        .orElseThrow(IllegalArgumentException::new).getFirstPaymentDate());
+                default:
+                    throw new IllegalArgumentException("Invalid part admission payment option");
```

Comment:
Default statement message is wrong - Invalid payment option is good enough..not neccesarily part admission.